pymc3/tests/test_distributions.py

```diff
@@ -1183,6 +1183,10 @@ def test_bound():
     assert rand.dtype in [np.int16, np.int32, np.int64]
     assert rand >= 5 and rand <= 8
 
+    with pm.Model():
+        BoundPoisson = pm.Bound(pm.Poisson, upper=6)
+        BoundPoisson(name="y", mu=1)
```

Comment:
Hmmmmmmmmm we never do this... it is always `BoundPoisson("y", mu=1)`

Reply:
(You must have been playing around with TFP and Edwards2 lately LOL)

Reply:
Wasn't me :-)
I think an explicit `name` should work though. That is just the norm in python.

Reply:
I see. We do have the same pattern in FreeRV etc

Reply:
This works:
```
with pm.Model() as model:
    pm.Normal(name='a')
    pm.Normal(name='b', observed=[0.])
```